client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import InlineSupportLink from 'components/inline-support-link';
+import { localizeUrl } from 'lib/i18n-utils';
+import isClassicEditorForced from 'state/selectors/is-classic-editor-forced';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+const MasteringGutenberg = ( { cannotUseGutenberg, translate } ) => {
+	if ( cannotUseGutenberg ) {
+		return null;
+	}
+
+	return (
+		<Card className="mastering-gutenberg">
+			{ ! isMobile() && (
+				<div className="mastering-gutenberg__illustration">
+					<img src="/calypso/images/illustrations/gutenberg-mini.svg" alt="" />
+				</div>
+			) }
+			<div>
+				<CardHeading>{ translate( 'Master the Block Editor' ) }</CardHeading>
+				<p className="mastering-gutenberg__text customer-home__card-subheader">
+					{ translate(
+						'Learn how to create stunning post and page layouts ' + 'through our video guides.'
```

Comment:
Seems we don't need to concat the 2 parts here.
```suggestion
						'Learn how to create stunning post and page layouts through our video guides.'
```

Reply:
Hm, not sure why I did that to begin with. Fixed. :)